nifi-nar-bundles/nifi-snmp-bundle/nifi-snmp-processors/src/main/java/org/apache/nifi/snmp/dto/SNMPSingleResponse.java

```diff
@@ -65,4 +68,17 @@ public int getVersion() {
     public boolean isReportPdu() {
         return responsePdu.getType() == PDU.REPORT;
     }
+
+    public boolean isOidProvidedByFlowFile() {
+        return !incomingFlowFileAttributes.isEmpty();
+    }
+
+    public Map<String, String> getOutgoingFlowFileAttributes() {
```

Comment:
It's probably best not to filter out any of the incoming attributes.
That being said, the response shouldn't handle the incoming attributes anyway. That responsibility is better left at the processor level.